api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -1436,9 +1436,10 @@ public Allergies setAllergies(Patient patient, Allergies allergies) {
 			if (allergy.getAllergyId() == null && allergy.getAllergen().getCodedAllergen() == null
 			        && StringUtils.isNotBlank(allergy.getAllergen().getNonCodedAllergen())) {
 				
-				Concept otherNonCoded = Context.getConceptService().getConceptByUuid(Allergen.OTHER_NON_CODED_UUID);
+				String otherNonCodedConceptUuid = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_ALLERGEN_OTHER_NON_CODED_UUID);
```

Comment:
Since we set this on startup, do we need to fetch it again instead of just using Allergen. getOtherNonCodedConceptUuid()?